src/utils/parsers/__tests__/currency.test.js

```diff
@@ -0,0 +1,25 @@
+import { centsToReal, realToCents } from 'utils/parsers/currency';
+
+describe('Currency parsers', () => {
+  describe('centsToReal', () => {
+    it('should only accept an Integer numbers', () => {
+      expect(() => centsToReal('123')).toThrow(TypeError);
+      expect(() => centsToReal(123.1)).toThrow(TypeError);
+    });
```

Comment:
Since you've changed the `centsToReal` function to not throw errors anymore, I think these tests should fail, don't they?